dir.props

```diff
@@ -1,10 +1,17 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition=" Exists('obj/Microsoft.DotNet.Cli.BuildInfo.props') "
+          Project="obj/Microsoft.DotNet.Cli.BuildInfo.props" />
+  <Import Condition=" Exists('obj/Microsoft.DotNet.Cli.HostInfo.props') "
+          Project="obj/Microsoft.DotNet.Cli.HostInfo.props" />
   <Import Project="build/Microsoft.DotNet.Cli.DependencyVersions.props" />
 
   <!-- NuGet Information -->
   <PropertyGroup>
     <RepoRoot>$(MSBuildThisFileDirectory)</RepoRoot>
-    
+
+    <BuildInfoProps>$(RepoRoot)/obj/BuildInfo.props</BuildInfoProps>
```

Comment:
No matter where we put these files, we should have a single property for the directory and use that property everywhere.  Not having "obj" duplicated so many times.